code/greedy_algorithms/src/minimum_coins/minimum_coins.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+
+int 
+main()
+{
+	printf("Enter Number of Denominations \n");
+	int n;
+	scanf("%d", &n);
+	int denominations[n], frequency[n];
+	for(int i = 0; i < n; i++)
```

Comment:
You may want to move i outside the for loop for compatibility with older C versions.